packages/orchestrator-ui-components/src/components/WfoTable/WfoBasicTable/styles.ts

```diff
@@ -48,8 +48,34 @@ export const getStyles = (theme: EuiThemeComputed) => {
             cursor: isSortable ? 'pointer' : 'not-allowed',
         });
 
+    const basicTableWithStatusColorColumn = css(
+        {
+            '.wfoBasicTable thead td': {
+                paddingBlock: 8,
+                paddingLeft: 8,
+            },
+            '.wfoBasicTable thead th': {
+                paddingBlock: 8,
+                paddingLeft: 8,
+            },
+            '.wfoBasicTable thead th button span:first-child': {
+                padding: 0,
+                border: 0,
+            },
+            '.wfoBasicTable tbody tr td div:first-child': {
+                padding: 0,
+                border: 0,
+            },
+            '.wfoBasicTable thead svg:not(.wfoArrowNarrowIcon)': {
```

Comment:
It is using the Eui table sorting, so it shows by default the EuiSortingIcon. That is the default behaviour of the EuiBasicTable. This will hide the Eui version and replace it with our icon.